### PR TITLE
Add support for github-ci and prepare releasing on PyPi

### PR DESCRIPTION
**Pull request by adybbroe**
Add support for github-ci and add admin for standard releasing procedures

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<!-- Describe what your PR does, and why -->
+
+ - [ ] Closes #xxxx <!-- remove if there is no corresponding issue, which should only be the case for minor changes -->
+ - [ ] Tests added <!-- for all bug fixes or enhancements -->
+ - [ ] Tests passed: Passes ``pytest pyspectral`` <!-- for all non-documentation changes) -->
```
Comment (ninahakansson) [inline .github/PULL_REQUEST_TEMPLATE.md]: pyspectral => level1c4pps

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+ - [ ] Passes ``flake8`` <!-- remove if you did not edit any Python files -->
```
Comment (ninahakansson) [inline .github/PULL_REQUEST_TEMPLATE.md:6]: Suggestion: git diff origin/master -- "*py" | flake8 --diff

File: RELEASING.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+3. run the unittests
+4. run `loghub` and update the `CHANGELOG.md` file:
+
+```
+loghub foua-pps/level1c4pps --token <personal access token (see https://github.com/settings/tokens)>  -st v<previous version> -plg bug "Bugs fixed" -plg enhancement "Features added" -plg documentation "Documentation changes"
+```
+
+Don't forget to commit!
+
+5. Create a tag with the new version number, starting with a 'v', eg:
```
Comment (ninahakansson) [inline RELEASING.md:17]: Suggestion: Create a tag with bumpversion i.e: bumpversion patch

File: RELEASING.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+```
+git tag -a v0.1.1 -m "Version 0.1.1"
+```
+
+See [semver.org](http://semver.org/) on how to write a version number.
```
Comment (ninahakansson) [inline RELEASING.md:23]: Can be removed as we use bumpversion?

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,2 @@
+## Version <v0.2.2> (2021/05/19)
```
Comment (ninahakansson) [inline CHANGELOG.md]: Level1c4pps is currently at v0.2.25?

File: RELEASING.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+# Releasing activefires-pp
+
+prerequisites: `pip install setuptools twine`
+
+
+1. checkout main
```
Comment (ninahakansson) [inline RELEASING.md]: Should it be master?

File: RELEASING.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
+# Releasing activefires-pp
```
Comment (ninahakansson) [inline RELEASING.md]: activefires-pp => level1c4pps

**Review by ninahakansson — COMMENTED** [review]
Thanks @adybbroe nice work! Some of it I do not understand yet, but I made some comments.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (adybbroe) [reply .github/PULL_REQUEST_TEMPLATE.md:6]: @ninahakansson running just plainly `flake8` is fine I would say.
It urge everyone to fix and get rid of *all* flake8 complaints, not only the ones you introduce yourself

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (adybbroe) [reply .github/PULL_REQUEST_TEMPLATE.md]: Thanks. Fixed!

File: RELEASING.md (hunk 4 above)
Comment (adybbroe) [reply RELEASING.md:23]: I rather keep it. We only tag when preparing for a PyPi release, and when we have seen that the github actions are green. I am personally not using bumpversion anymore. As I don't think it is adding so much now that we do versioning using the `setupttools_scm` package. That way we don't need to have the version stored anywhere (in a python file like `version-py`)

File: RELEASING.md (hunk 6 above)
Comment (adybbroe) [reply RELEASING.md]: Yes, correct. But we should change to use `main` for the main branch as is standard nowadays. But we fix this indpependently of this one.

File: RELEASING.md [hunk 8]
```diff
@@ -0,0 +1,37 @@
...
+
+See [semver.org](http://semver.org/) on how to write a version number.
+
+
+6. push changes to github `git push --follow-tags`
+
+7. Verify the Github actions unit tests passed
+
+8. Create a "Release" on GitHub by going to
+   https://github.com/adybbroe/activefires-pp/releases and clicking "Draft a new
```
Comment (ninahakansson) [inline RELEASING.md]: adybbroe/activefires-pp => foua-pps/level1c4pps

File: RELEASING.md (hunk 4 above)
Comment (ninahakansson) [reply RELEASING.md:23]: Let's try without bumpversion!

File: RELEASING.md (hunk 8 above)
Comment (adybbroe) [reply RELEASING.md]: Incredible! Thanks! :-)
Fixed

**Pull request merged**